.vscode/settings.json

```diff
@@ -67,19 +95,33 @@
     // Files
     //////////////////////////////////////
     "files.exclude": {
-        "node_modules/": true,
+        "node_modules/": false,
         "pdfcomprezzor/": true,
         "coverage/": true,
         "solr/": true,
         ".docker/": true
     },
     "files.associations": {
+        "*.erb": "erb",
```

Comment:
In one of the latest commits, I also changed this line, which means that `.html.erb` files are now identified as ERB language and not as HTML language. This allows us to also have Ruby syntax highlighting inside those files, while HTML syntax highlighting still works. Linting itself is unaffected by this setting.

Also note the comment following this line, which explains why we still use `javascript` and not `erb` as language option for `.js.erb` files (TL;DR: JavaScript is not correctly syntax-highlighted in erb files otherwise).